blocks/columns/columns.js

```diff
@@ -15,4 +15,22 @@ export default function decorate(block) {
       }
     });
   });
+
+  if (block.classList.contains('collections')) {
+    block.parentElement.classList.add('column-collections-parent');
+    // query anchor tags in the block and move the picture into the anchor
+    const anchors = [...block.querySelectorAll('a')];
+    anchors.forEach((anchor) => {
+      const h2 = document.createElement('h2');
+      h2.classList.add('columns-img-header');
+      h2.innerHTML = anchor.textContent.toUpperCase();
+      anchor.text = '';
+      anchor.append(h2);
+
+      const h1 = anchor.parentElement;
```

Comment:
you can make this slight more robust by doing something like
```suggestion
      const header = anchor.closest('h1, h2, h3, h4, h5, h6');
      anchor.append(header.queryElement('picture'))
      header.replaceWith(anchor);
```
then any header can be used, and the position of the picture can be flexible.

Reply:
`replaceWith` will help remove the extra h2: 
![Screenshot 2023-08-21 at 16 53 37](https://github.com/hlxsites/24life/assets/2250964/a4eff82b-79f7-4797-a82b-78ecf70d4a05)